IQSS-Dataverse-Develop.xml

```diff
@@ -46,38 +46,55 @@
 echo &quot;exit on error&quot;
 set -e
 
-echo &quot;package warfile&quot;
-mvn -DcompilerArgument=-Xlint:unchecked test -P all-unit-tests package</command>
-    </hudson.tasks.Shell>
-    <hudson.tasks.Shell>
-      <command>#!/usr/bin/env bash
-
-echo &quot;Setting job to exit on error...&quot;
-set -e
+echo &quot;start with a clean plate&quot;
+rm -rf target/
 
-echo &quot;Installing Dataverse in Docker using docker-aio...&quot;
-conf/docker-aio/prep_it.bash
-
-echo &quot;Running API test suite using docker-aio...&quot;
-conf/docker-aio/run-test-suite.sh
-
-echo &quot;copying server.log to conf/docker-aio/&quot;
-docker cp dv:/usr/local/glassfish4/glassfish/domains/domain1/logs/server.log conf/docker-aio/</command>
-    </hudson.tasks.Shell>
-    <hudson.tasks.Shell>
-      <command>#!/usr/bin/env bash
+echo &quot;package warfile&quot;
+mvn -DcompilerArgument=-Xlint:unchecked test -P all-unit-tests package
 
-mvn test -Dtest=DataversesIT,DatasetsIT,SwordIT,AdminIT,BuiltinUsersIT,UsersIT,UtilIT,ConfirmEmailIT,FileMetadataIT,FilesIT,SearchIT,InReviewWorkflowIT,HarvestingServerIT,MoveIT,MakeDataCountApiIT -Ddataverse.test.baseurl=http://localhost:8080</command>
+echo &quot;generate HTML SureFire output&quot;
+mvn surefire-report:report</command>
     </hudson.tasks.Shell>
   </builders>
   <publishers>
+    <org.jenkinsci.plugins.postbuildscript.PostBuildScript plugin="postbuildscript@2.9.0">
+      <config>
+        <scriptFiles/>
+        <groovyScripts/>
+        <buildSteps>
+          <org.jenkinsci.plugins.postbuildscript.model.PostBuildStep>
+            <results>
+              <string>SUCCESS</string>
+            </results>
+            <role>BOTH</role>
+            <buildSteps>
+              <hudson.tasks.Shell>
+                <command>curl -O https://raw.githubusercontent.com/IQSS/dataverse-ansible/master/ec2/ec2-create-instance.sh</command>
+              </hudson.tasks.Shell>
+            </buildSteps>
+          </org.jenkinsci.plugins.postbuildscript.model.PostBuildStep>
+          <org.jenkinsci.plugins.postbuildscript.model.PostBuildStep>
+            <results>
+              <string>SUCCESS</string>
+            </results>
+            <role>BOTH</role>
+            <buildSteps>
+              <hudson.tasks.Shell>
+                <command>bash ec2-create-instance.sh -b develop -r https://github.com/IQSS/dataverse.git -t jenkins_delete_me -l target -g groupvars.yml</command>
```

Comment:
It looks like this is the next thing to fix:

ec2-create-instance.sh: line 110: aws: command not found
The "aws" program could not be executed. Is it in your $PATH?